### PR TITLE
Added documentation for the new GCP stackdriver scaler

### PR DESCRIPTION
**Pull request by RamCohen**
Added documentation for the new GCP stackdriver scaler

Relates to https://github.com/kedacore/keda/issues/2661

Signed-off-by: Ram Cohen <ram.cohen@gmail.com>

### CODE REVIEW

File: content/docs/2.7/scalers/gcp-stackdriver.md
Comment (zroubalik) [inline content/docs/2.7/scalers/gcp-stackdriver.md:20]: Can we please not force users to specify `metricName`, we should generate it instead.

File: content/docs/2.7/scalers/gcp-stackdriver.md
Comment (zroubalik) [inline content/docs/2.7/scalers/gcp-stackdriver.md:35]: I don't see this anywhere used. What is the purpose anyway?

File: content/docs/2.7/scalers/gcp-stackdriver.md
Comment (RamCohen) [reply content/docs/2.7/scalers/gcp-stackdriver.md:20]: I know but there isn't any other parameter I can deduce a reasonable metric from. project_id is too generic and can collide with another such scaler

File: content/docs/2.7/scalers/gcp-stackdriver.md
Comment (zroubalik) [inline content/docs/2.7/scalers/gcp-stackdriver.md:30]: I think you can omit this line, it is just internal detail. Not user facing.

**Review by zroubalik — APPROVED** [review]
LGTM

/hold for others before merging.

**Review by JorTurFer — APPROVED** [review]
LGTM!
Thanks for this! ❤️

**Pull request merged**
